.github/workflows/CI.yml

```diff
@@ -737,31 +749,35 @@ jobs:
             "$HOME/Library/Application Support/com.apple.TCC/TCC.db"
           )
 
-          sqlQuery="INSERT OR IGNORE INTO access VALUES($values);"
-
-          for dbPath in "${dbPaths[@]}"; do
-            echo "Column names for $dbPath"
-            echo "-------------------"
-            sudo sqlite3 "$dbPath" "PRAGMA table_info(access);"
-            echo "Current permissions for $dbPath"
-            echo "-------------------"
-            sudo sqlite3 "$dbPath" "SELECT * FROM access WHERE service='kTCCServiceScreenCapture';"
-            sudo sqlite3 "$dbPath" "$sqlQuery"
-            echo "Updated permissions for $dbPath"
-            echo "-------------------"
-            sudo sqlite3 "$dbPath" "SELECT * FROM access WHERE service='kTCCServiceScreenCapture';"
+          for value in "${values[@]}"; do
+            for dbPath in "${dbPaths[@]}"; do
+              echo "Column names for $dbPath"
+              echo "-------------------"
+              sudo sqlite3 "$dbPath" "PRAGMA table_info(access);"
+              echo "Current permissions for $dbPath"
+              echo "-------------------"
+              sudo sqlite3 "$dbPath" "SELECT * FROM access WHERE service='kTCCServiceScreenCapture';"
+              execute_sql_query "$value" "$dbPath"
+              echo "Updated permissions for $dbPath"
+              echo "-------------------"
+              sudo sqlite3 "$dbPath" "SELECT * FROM access WHERE service='kTCCServiceScreenCapture';"
+            done
           done
 
       - name: Run tests
         id: test
         timeout-minutes: 10
+        working-directory:
+          /opt/local/var/macports/build/_Users_runner_work_Sunshine_Sunshine_ports_multimedia_Sunshine/Sunshine/work/build/tests
         run: |
-          sudo port test "Sunshine"
+          sudo port install doxygen
+          sudo port install graphviz
+          sudo ./test_sunshine --gtest_color=yes
 
       - name: Test Logs
         if: always()
         run: |
-          logfile="/opt/local/var/macports/logs/_Users_runner_work_Sunshine_Sunshine_ports_multimedia_Sunshine/Sunshine/main.log"
+          logfile="/opt/local/var/macports/build/_Users_runner_work_Sunshine_Sunshine_ports_multimedia_Sunshine/Sunshine/work/build/tests/test.log"
           cat "$logfile"
```

Comment:
We don't need test logs anymore if running the test binary manually.

Reply:
These logs are the boost_log outputs of the code run during the tests, without the Test logs, just pure execution logs, felt useful to me, but I can remove.